book/scheduling.md

```diff
@@ -562,8 +560,8 @@ if __name__ == "__main__":
 Now we're scheduling a new rendering task every 16 milliseconds, just
 as we wanted to.
 
-Optimizing rendering tasks
-==========================
+Optimizing rendering
```

Comment:
This avoids line wrapping.

Reply:
In my PR I change the title to "Optimizing with dirty bits", take your pick of which you like more.